src/docs/devices/Elecrow-5inch-ESP32-display/index.md

```diff
@@ -0,0 +1,402 @@
+---
+title: CrowPanel 5.0" HMI ESP32 display (by Elecrow)
+date-published: 2024-08-17
+type: misc
+standard: global
+project-url: https://www.elecrow.com/wiki/esp32-display-502727-intelligent-touch-screen-wi-fi26ble-800480-hmi-display.html
+board: esp32
+---
+
+![Product Image](elecrow-5-inch-esp32-display.jpg "Product Image")
+
+## Drivers
+
+* Processor: ESP32-S3-WROOM-1-N4R8 (using `esp32-s3-devkitc-1` board)
+* Touchscreen: `gt911`
+* Display: ILI6122 & ILI5960 (using `rpi_dpi_rgb` plaform)
+
+## GPIO Pinout
+
+### i2c (used for touchscreen)
+
+| Pin    | Function      |
+| ------ | ------------- |
+| GPIO19 | i2c SDA     |
+| GPIO20 | i2c SCL     |
+
+### backlight
+
+| Pin    | Function      |
+| ------ | ------------- |
+| GPIO2  | backlight   |
+
+### display
+
+| Pin    | Function      |
+| ------ | ------------- |
+| GPIO40 | de_pin      |
+| GPIO39 | hsync_pin   |
+| GPIO41 | vsync_pin   |
+| GPIO0  | pclk_pin    |
+
+### data pins
+
+```yaml
+red:
+  - 45        #r1
+  - 48        #r2
+  - 47        #r3
+  - 21        #r4
+  - 14        #r5
+green:
+  - 5         #g0
+  - 6         #g1
+  - 7         #g2
+  - 15        #g3
+  - 16        #g4
+  - 4         #g5
+blue:
+  - 8         #b1
+  - 3         #b2
+  - 46        #b3
+  - 9         #b4
+  - 1         #b5
+```
+
+## Simle configuration
+
+```yaml
+esphome:
+  name: display
+  platformio_options:
+    build_flags: "-DBOARD_HAS_PSRAM"
```

Comment:
Not required as this is set automatically with the `psram` component when using Arduino
```suggestion
```